Add meta tags

cveneziani opened the pull request:
- Add Images, Meta Tags and URL helpers
- Add description to all pages
- Namespace images per page path

flexbox commented on source/layouts/shared/_meta_tags.html.slim:
```diff
@@ -0,0 +1,35 @@
+meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"
+meta name="viewport"              content="width=device-width, initial-scale=1.0"
+
+meta name="description"           content=meta_tag_description
```
`content="#{meta_tag_description}"`

flexbox replied:
why the `title` is not here ?

cveneziani replied:
Because it's not a `meta` tag? :grimacing:

flexbox commented on source/layouts/shared/_meta_tags.html.slim:
```diff
@@ -0,0 +1,35 @@
+meta name="geo.region"            content="FR-59"
+
+/ Microdata
+meta itemprop="name"              content=meta_tag_title
```
same as above

cveneziani committed:
Add meta tags

- Add Images, Meta Tags and URL helpers
- Add description to all pages
- Move page title to meta tags partial
- Namespace images per page path
